docs/samples/deployments/k8s/index.md

```diff
@@ -0,0 +1,74 @@
+# Deploy presidio to Kubernetes
+
+You can install Presidio locally using [KIND](https://github.com/kubernetes-sigs/kind), as a service in [Kubernetes](https://kubernetes.io/) or [AKS](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes).
+
+
+- [Deploy locally using KIND](#deploy-locally-with-kind)
+- [Deploy with Kubernetes](#presidio-as-a-service-with-kubernetes)
+  - [Prerequisites](#prerequisites)
+  - [Step by Step Deployment with customizable parameters](#step-by-step-deployment-with-customizable-parameters)
+
+## Deploy locally with KIND
+
+[KIND (Kubernetes IN Docker)](https://github.com/kubernetes-sigs/kind).
+
+1. Install [Docker](https://docs.docker.com/install/).
+
+2. Clone Presidio.
+
+3. Run the following script, which will use KIND (Kubernetes emulation in Docker)
+
+   ```sh
+   cd deployment/
+   ./run-with-kind.sh
+   ```
+
+4. Wait and verify all pods are running:
+
+   ```sh
+   kubectl get pod -n presidio
+   ```
+
+5. Port forwarding of HTTP requests to the API micro-service will be done automatically. In order to run manual:
+   ```sh
+   kubectl port-forward <presidio-analyzer-pod-name> 8080:8080 -n presidio
+   ```
+
+## Presidio As a Service with Kubernetes
+
+### Prerequisites
+
+1. A Kubernetes 1.9+ cluster with [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) enabled. If you are using [AKS](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes) RBAC is enabled by default.
```

Comment:
1.18+
same for the client, kind and helm versions should be updated to the recent ones.